src/rollbar.php

```diff
@@ -476,7 +464,8 @@ protected function _report_php_error($errno, $errstr, $errfile, $errline) {
         $payload = $this->build_payload($data);
 
         // Determine whether to send the request to the API.
-        if ($this->_shouldIgnore(true, new RollbarException($level, $errstr), $payload)) {
+        $exception = new ErrorException($error_class, 0, $errno, $errfile, $errline);
```

Comment:
:+1: That's great! I didn't know `ErrorException` existed. That's a nice solution.